### PR TITLE
Feature/get user by id

### LINKED ISSUE #2: Create endpoint to GET a User by Id
API endpoint to allow GET by user ID. 

JSON property is "id" and is a number. 

If invalid ID requested, send JSON Response {"error": "Specified User not found"}

### PR DESCRIPTION
**Pull request by Kiresorg**
Please review. Closes #2

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -10,5 +10,15 @@ router.get("/users", async (req, res) => {
   res.send(users);
 });
 
+router.get("/users/:id", async (req, res) => 
+{
+  const userid = parseInt(req.params.id);
```
Comment (m-labbe) [inline routes/users.js]: Can we use camelCase for this variable name?

File: routes/users.js [hunk 2]
```diff
@@ -10,5 +10,15 @@ router.get("/users", async (req, res) => {
...
+  if(isNaN(userid)) 
+  {
+    res.send({"error": "Specified User not found"});
+  }
+  var user = users.find(item => item.id === userid);
```
Comment (m-labbe) [inline routes/users.js]: Let's make this a `const`

**Review by m-labbe — CHANGES_REQUESTED** [review]
Look good! Can you add some tests for this endpoint?

File: routes/users.js (hunk 1 above)
Comment (Kiresorg) [reply routes/users.js]: Done

File: routes/users.js (hunk 2 above)
Comment (Kiresorg) [reply routes/users.js]: Done

**Comment by Kiresorg** [comment]
Made those changes and added tests. Great feedback. I also noticed I was gong to merge into main which would be bad :) I changed it to release branch.

**Review by m-labbe — APPROVED** [review]
LGTM!

**Pull request merged**